app/views/devise/registrations/new.html.haml

```diff
@@ -0,0 +1,24 @@
+.container
+  .row.justify-content-center
+    .col-lg-4.col-md-4
```

Comment:
Do you really need 'col-lg' class if you set 'col-md' class and they have equal number of columns?